src/output.rs

```diff
@@ -1,39 +1,77 @@
-use std::collections::HashMap;
-
-use frost_ed25519 as frost;
-
-use frost::keys::{KeyPackage, PublicKeyPackage};
+use frost::keys::{PublicKeyPackage, SecretShare, VerifiableSecretSharingCommitment};
 use frost::Identifier;
+use frost_ed25519 as frost;
 use itertools::Itertools;
+use std::collections::HashMap;
 
 pub trait Logger {
     fn log(&mut self, value: String);
 }
 
+fn encode_commitment(vss_commitment: VerifiableSecretSharingCommitment) -> String {
```

Comment:
The parameter can be a reference (`&VerifiableSecretSharingCommitment`), this will avoid having the callers having to `clone()` it

Reply:
Completed in [0ca8ddc](https://github.com/ZcashFoundation/frost-zcash-demo/pull/10/commits/0ca8ddc076f4c24464c8c204974f07ebfadc3f05)